.github/workflows/merge.yml

```diff
@@ -282,6 +282,12 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 5
 
+      - name: Start a client to upload cost estimate
+        run: cargo run --bin safe --release -- --log-output-dest=data-dir files estimate "./resources"
```

Comment:
is this happening? CI is all green at the moment.

`to pass '--retry-strategy' as a value, use '-- --retry-strategy'` is what you should be using here. If there's a need for `retry_strategy` (cc @RolandSherwin is there?), then we'll need to get it in and see the real error from that, I think.

Reply:
>> cargo run --bin safe --release -- --log-output-dest=data-dir files estimate "./resources" -- --retry-strategy quick

Finished release [optimized] target(s) in 0.61s
     Running `target\release\safe.exe --log-output-dest=data-dir files estimate ./resources -- --retry-strategy quick`
error: unexpected argument '--retry-strategy' found

Reply:
so does that argument exist? I dont think it does for `estimate` ?

Reply:
It seems the retry strategy is set to try the request again in case of file corruption during upload/download.
I don't think it's strictly necessary for an estimate. @RolandSherwin can you confirm?

Reply:
It could be. But I mean, that isn't present on your argument setup for the `estimate` command in the CLI, so that's why you're seeing that error

Reply:
ie, its no here: https://github.com/maidsafe/safe_network/pull/1321/files#diff-e2769aa51eabf52db346b913959f5906a8f1093872adaa8559924bc135c439a3R59-R63

Reply:
yes. I see estimate uses a different api.
retry-strategy is part of: 

FilesUpload::new(files_api)
        .set_batch_size(batch_size)
        .set_verify_store(verify_store)
        .set_retry_strategy(retry_strategy);

which estimate does not use. So it does not need to be included.

Reply:
The `RetryStrategy` should be manually implemented for `get_store_cost`. We only have the for `get_record` and `put_record` I'm afraid. 
I feel we can ignore the `--retry-strategy` flag on `estimate` for now. The `get_store_cost` function waits for response from 5 peers and even if we get 1 response back, we will not be erroring out. So this feature should work without trouble under most circumstances.